bundles/mapping/mapwfs2/plugin/components/ReqEventHandler.js

```diff
@@ -60,6 +112,34 @@ export default class ReqEventHandler {
                     const propsList = OLLayer.getSource().getPropsIntersectingGeom(filterFeature.geometry);
                     modifySelection(layer, propsList.map(props => props[WFS_ID_KEY]), keepPrevious);
                 });
+            },
+            'WFSSetPropertyFilter': event => {
+                if (!event.getFilters() || event.getFilters().filters.length === 0) {
+                    return;
+                }
+                const layer = getSelectedLayer(event.getLayerId());
+                if (!layer) {
+                    return;
+                }
+                const records = layer.getActiveFeatures();
+                if (!records || records.length === 0) {
+                    return;
+                }
+                const fields = layer.getFields();
+                const idIndex = fields.indexOf(WFS_FTR_ID_KEY);
+                let filteredIds = [];
+                // handle filter order (AND/OR)
+                const filterOrder = orderFilterOperations(event.getFilters().filters);
```

Comment:
`orderFilterOperations` returns a list of OR groups. We could reflect this in variable naming. So when we do `.forEach` later it will be clear what we are iterating over.